Tests/Binary/Loader/AbstractDoctrineLoaderTest.php

```diff
@@ -34,12 +36,16 @@ class AbstractDoctrineLoaderTest extends TestCase
 
     public function setUp(): void
     {
-        if (!interface_exists(ObjectManager::class)) {
+        if (interface_exists(LegacyObjectManager::class)) {
+            $omClassName = LegacyObjectManager::class;
+        } elseif (interface_exists(ObjectManager::class)) {
+            $omClassName = ObjectManager::class;
+        } else {
             $this->markTestSkipped('Requires the doctrine/orm package.');
```

Comment:
As `doctrine/orm` is in `require-dev` it will always be installed when running tests.Therefore you can remove the `else` statement.

Reply:
done.